src/k-provider/ovp/provider.js

```diff
@@ -74,11 +88,19 @@ export default class OVPProvider extends BaseProvider<OVPProviderMediaInfoObject
     });
   }
 
-  doRequest(loaders: Array<RequestLoader>): Promise<any> {
-    const dataLoader = new OVPDataLoaderManager(this.playerVersion, this.partnerId, this.ks, this._networkRetryConfig);
+  doRequest(loaders: Array<RequestLoader>, ks?: string): Promise<any> {
+    let theKs: string = ks || this.userKs;
```

Comment:
need to save the ks we get from the app as `this.ks`  (same logic we have in getMediaConfig)

Reply:
done